Linux/flash

```diff
@@ -207,11 +212,8 @@ while true; do
     esac
   done
 
-  # paranoia sync before umounting
-  echo "Synchronizing Filesystems"
-  sudo sync; sudo sync; sudo sync
-
-  writable=$(grep "${disk}" /etc/mtab | grep -e "\( rw,\|,rw\)")
+  $(sudo test -w "${disk}")
```

Comment:
Unfortunately, *write protected* here is a mount option, so we don't know if the drive is mounted until we mount it.

Tell me what you think about this: instead of checking if the drive is writable, we could instead do *mount -o remount,rw ${disk}* to remount as read-write. As a safety check, we could also prompt for user input to make sure the user is OK with it.